README.md

```diff
@@ -1,7 +1,7 @@
 9wm
 ====
 
-9wm is an X11 window manager inspired by the Plan 9 window manager 8½.
+9wm is an X11 window manager inspired by the Plan 9 window manager rio.
```

Comment:
I think just change it back.  ☺

On Wed, Feb 24, 2016, 17:57 Jacob Adams <notifications@github.com> wrote:

> In README.md <https://github.com/nealey/9wm/pull/1#discussion_r54033992>:
>
> > @@ -1,7 +1,7 @@
> >  9wm
> >  ====
> >
> > -9wm is an X11 window manager inspired by the Plan 9 window manager 8½.
> > +9wm is an X11 window manager inspired by the Plan 9 window manager rio.
>
> Ah ok Would you rather I change it back or just change it to "8 1/2 / rio"
> or something like that?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nealey/9wm/pull/1/files#r54033992>.
>